build.gradle

```diff
@@ -289,6 +328,15 @@ allprojects {
 //
 // Command line args can be passed to the tool as a comma-separated list:
 //   ./gradlew runVGen '-Pclargs=--arg1=val1,--arg2=val2,--arg3=val3'
+task runCodeGen(type: JavaExec) {
+  classpath = sourceSets.main.runtimeClasspath
+  main = 'com.google.api.codegen.CodeGeneratorTool'
+  if (project.hasProperty('clargs')) {
+    args = clargs.split(',').toList()
+  }
+}
+
+// old task being retained for backwards-friendliness
```

Comment:
It should just be artman, but I think it would be less frustrating to the team in general if artman/toolkit can be updated at the leisure of each dev.